README.md

```diff
@@ -1,5 +1,7 @@
 # Tagua VM [![build status](https://api.travis-ci.org/tagua-vm/tagua-vm.svg)](https://travis-ci.org/tagua-vm/tagua-vm)
```

Comment:
I would rather put one badge per line, so please @jubianchi, can you put Travis CI badge on its own line. Also, you can change “build status” for “Build status”. Same for “Join the chat at…” by “Chat on Gitter” for instance.